vit-servicing-station-lib/src/server/signals.rs

```diff
@@ -0,0 +1,25 @@
+use tokio::signal;
+
+#[cfg(not(target_os = "windows"))]
```

Comment:
I think this should be the other way around: this function's body is dependent on target_os = "unix", whereas the Windows variant is actually cross-platform, so it should be a fallback for non-UNIX.